src/reason_parser.mly

```diff
@@ -4095,12 +4228,16 @@ val_ident:
   | PERCENT                                     { "%" }
   | LESSGREATER                                 { "<>" }
   | LESSDOTDOTGREATER                           { "<..>" }
+  | SLASHGREATERLESS                            { "/><" }
```

Comment:
Yeah, I recall that and I think it should be documented somewhere why <X /><Y> would not parse while the parse rules are in terms of tokens `<` `X` `/>` `<` `Y` `>`.